### PR TITLE
Create addon ns when it is not the default addon ns

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: pkg/addon/manifests/chart/templates/namespace.yaml [hunk 1]
```diff
@@ -1,12 +1,16 @@
-{{- if eq .Values.installMode "Hosted" }}
+# namespace is klusterlet-{clusterName} in hosted mode, and we need to
+# set specific label/annotation.
+{{- if ne .Release.Namespace "open-cluster-management-agent-addon" }}
 apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Release.Namespace }}
   labels:
+  {{- if eq .Values.installMode "Hosted" }}
```
Comment (zhujian7) [inline pkg/addon/manifests/chart/templates/namespace.yaml:9]: does this also need label `addon.open-cluster-management.io/namespace` like https://github.com/open-cluster-management-io/managed-serviceaccount/pull/109

Comment (qiujian16) [reply pkg/addon/manifests/chart/templates/namespace.yaml:9]: this label has been added I think

**Pull request merged**